docs/source/install.rst

```diff
@@ -5,44 +5,70 @@
 Installation
 ============
 
-The following is a quick start overview of installing the Jupyter Notebook and kernels. For detailed installation
-instructions for individual Jupyter or IPython projects, please see the documentation and/or GitHub repos
-of the individual projects listed :ref:`here <landing>`.
+The following is a quick start overview of installing the Jupyter Notebook and
+kernels. For detailed installation instructions for individual Jupyter or IPython
+subprojects, please see the documentation and/or GitHub repos of the individual
+projects listed :ref:`here <subprojects>`.
 
 Installing the Jupyter Notebook
 -------------------------------
 
-While the Jupyter Notebook allows users to run code in many different programming
-language, the Notebook itself is implemented in Python.
+While the Jupyter Notebook allows users to run code in many different
+programming language, the Notebook itself is implemented in Python. Thus to
+install the Notebook you will also need Python.
 
-**If you already have Python**, you can install the notebook using pip::
-
-    pip install jupyter
 
+If you are new to Python
+........................
 
-**If you're new to Python**, we recommend installing `Anaconda
-<http://continuum.io/downloads#py34>`_, a Python distribution which includes
-Jupyter. After you install it, ensure that Jupyter is up to date by running::
+If you're new to Python we recommend installing `Anaconda
+<http://continuum.io/downloads#py34>`_, which includes Python itself, the Jupyter
+Notebook and other packges for scientific computing and data science. After you
+install it, ensure that Jupyter is up to date by running::
 
     conda update jupyter
```

Comment:
If anaconda package is installed and launcher used, the default that comes up is ipython-notebook v3 with an upgrade button next to it. This is the screenshot after clicking upgrade.
<img width="796" alt="screen shot 2015-09-03 at 1 50 48 pm" src="https://cloud.githubusercontent.com/assets/2680980/9670775/01f4fbdc-5243-11e5-9e6b-112d78eacd0b.png">

After launch, you do land on Jupyter. Is there an easy way to display Jupyter version from the web page or on the webpage?